libraries/rush-lib/src/schemas/subspace.schema.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
This file was called "subspaces.json" in the RFC - are we changing the name?

Reply:
changed